File: 10 - Python/02 - Estrutura Condicional/Exercício 08.py
#Exercício - Temperatura

escala = input("Digite a escala (F/C): ")

if escala == "F":
    F = float(input("Digite a temperatura em Fahrenheit: "))
    C = 5/9 * (F - 32)
    print("Temperatura em Celsius: ", round(C,2), "°C")
elif escala == "C":
    C = float(input("Digite a temperatura em Celsius: "))
    F = C * 9 / 5 + 32
    print("Temperatura em Fahrenheit: ", round(F,2), "°F")
else:
    print("Valor inválido!")

File: 06 - C/Exercícios/02 - Estrutura Condicional/Exercício 08.c
//Exercício - Temperatura

#include <stdio.h>
#include <string.h>

int main() {

    char escala;
    double F, C;

    printf("Voce vai digitar a temperatura em qual escala (C/F)? ");
    scanf("%c", &escala);

    if (escala == 'F'){
        printf("Digite a temperatura em Fahrenheit: ");
        scanf("%lf", &F);
        C = 5.0 / 9.0 * (F - 32.0);
        printf("Temperatura equivalente em Celsius: %.2lf", C);
    }
    else{
        printf("Digite a temperatura em Celsius: ");
        scanf("%lf", &C);
        F = C * 9.0 / 5.0 + 32.0;
        printf("Temperatura equivalente em Fahrenheit: %.2lf", F);
    }

    return 0;

}